frontend/src/components/common/Filter/components/FilterMultiPanel/components/FilterPanelList/style.ts

```diff
@@ -1,29 +1,45 @@
-import { makeStyles } from "@material-ui/core";
+import styled from "@emotion/styled";
+import { ListItemText } from "@material-ui/core";
+import { List, ListItem } from "czifui";
 import { PT_TEXT_COLOR } from "src/components/common/theme";
 
-/* eslint-disable sort-keys -- ignore object key order for style objects */
-export const useFilterPanelListStyles = makeStyles({
-  listItem: {
-    color: PT_TEXT_COLOR,
-    letterSpacing: "-0.1px",
-    lineHeight: "18px",
-    margin: 0 /* overrides margin from layout.css */,
-    padding: "7px 8px",
-    "&:hover": {
-      backgroundColor: "rgba(167, 182, 194, 0.3)",
-      /* mimics bp menu item pseudo-class hover background color */
-    },
-    "&:active": {
-      backgroundColor: "rgba(115, 134, 148, 0.3)",
-      /* mimics bp menu item pseudo-class active background color */
-    },
-  },
-  listItemText: {
-    display: "flex",
-    margin: 0,
-  },
-  sublist: {
-    marginLeft: 22,
-  },
-});
-/* eslint-enable sort-keys -- ignore object key order for style objects */
+export const PanelSublist = styled(List)`
+  margin-left: 22px;
+`;
+
+export const PanelListItem = styled(ListItem)`
+  && {
```

Comment:
Hey @tihuan what are your thoughts on this? I had to add the `&&` to obtain specificity over the SDS List Item styles. I'm wondering if there is a better way to do this?

Reply:
Ah thanks so much for flagging this, @frano-m ! That's my bad 😆 I don't recall why I added extra specificity in `ListItem` in SDS, so I just created a PR to correct it!

https://github.com/chanzuckerberg/sci-components/pull/201

Reply:
Oh @tihuan I've been wondering why I was having a wrestling match ... thank you sooo much 🙏🏽 ... Happy days!